docs/configuration.rst

```diff
@@ -143,6 +143,26 @@ Phinx currently supports the following database adapters natively:
 * `SQLite <http://www.sqlite.org/>`_: specify the ``sqlite`` adapter.
 * `SQL Server <http://www.microsoft.com/sqlserver>`_: specify the ``sqlsrv`` adapter.
 
+MySQL
+`````````````````
+
+Since version 0.4.0 you can define a table prefix and table suffix:
```

Comment:
Documentation should always refer to the current version, so there really shouldn't be any references to versions or "works since" notes as this is irrelevant info, and should be kept to changelogs.